layouts/shortcodes/pages/release.html

```diff
@@ -1,104 +1,99 @@
-<div class="jakarta-download-delivers container">
-  <h2 class="heading-underline heading-underline-extended text-center">Jakarta EE 8 Delivers</h2>
-  
-  <div class="row download-highlights margin-top-40">
-    <div class="col-sm-12 download-highlights-item">
-      <span class="featured-circle-outline"><img width="50" class="icon margin-top-10" src="images/list.svg"></span>
-      <h3>Full Compatibility with Java<sup>TM</sup>&nbsp;EE 8</h3>
-      <p>Fully compatible with the Java EE 8 specifications and Technology Compatibility Kits (TCKs)</p>
-    </div>
-    <div class="col-sm-12 download-highlights-item">
-      <span class="featured-circle-outline"><img width="50" class="icon margin-top-10" src="images/design-1.svg"></span>
-      <h3>Open Specifications</h3>
-      <p>Leveraging the <a href="/about/jesp/">Jakarta EE Specification Process</a>, an open source specification process that enables code-first development</p>
+<div class="features-and-benefits container">
+  <h2 class="text-center text-blue fw-600"><strong>JAKARTA EE9</strong></h2>
```

Comment:
Not sure if you need a space or not between EE and 9 ?

Reply:
Probably add a space would be better? The design PDF did not have space

Reply:
Maybe ask marketing to make sure. Jakarta EE 8 have a space where ever its written.